pyproject.toml

```diff
@@ -0,0 +1,91 @@
+[tool.black]
+target-version = ['py311']
+
+# This pyproject.toml is setup so it can be used with or without poetry and also
+# supports editable installs (PEP 660) without breaking IDE and linter inspection.
+
+# DEVELOPER INSTALL INSTRUCTIONS (assuming current directory is at the top of the project) :
+# Poetry:
+#   poetry install --extras dev
```

Comment:
My fault for not fixing the original comment in screenpy and screenpy_selenium.  This really SHOULD say `poetry install --extras dev_all`